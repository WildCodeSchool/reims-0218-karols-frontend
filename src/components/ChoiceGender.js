import React from "react"
import styles from "../styles/cardModel.css"
import { Card, CardTitle, CardBody, Button } from "reactstrap"

const ChoiceGender = ({ gender, image }) => {
  return (
    <div className="mb-4">
      <div className={styles.card}>
        <Card className="cardModel card" style={{}}>
          <div
            className="wrapper text-center"
            style={{
              backgroundImage: `url(${image})`,
              backgroundSize: "cover",
              backgroundPosition: "center",
              backgroundRepeat: "no-repeat"
            }}
          >
            <div className="data">
              <CardBody className="content">
                <CardTitle
                  className="city"
                  style={{
                    fontWeight: "500",
                    marginTop: "10px"
                  }}
                >
                  <p
                    style={{
                      color: "#FFF"
                    }}
                  >
                    {gender}
                  </p>
                </CardTitle>
                <Button
                  style={{
                    display: "block",
                    margin: "2em auto 1em",
                    textAlign: "center",
                    fontSize: "12px",
                    color: "#fff",
                    lineHeight: "1",
                    position: "relative",
                    fontWeight: "700",
                    marginBottom: "40px",
                    padding: "10px 20px"
                  }}
                >
                  SÃ©lectionner
                </Button>
              </CardBody>
            </div>
          </div>
        </Card>
      </div>
    </div>
  )
}

export default ChoiceGender
