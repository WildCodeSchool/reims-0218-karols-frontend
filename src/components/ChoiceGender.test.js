import React from "react"
import ReactDOM from "react-dom"
import ChoiceGender from "./ChoiceGender"
import renderer from "react-test-renderer"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<ChoiceGender />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders correctly", () => {
  const tree = renderer.create(<ChoiceGender />).toJSON()
  expect(tree).toMatchSnapshot()
})
