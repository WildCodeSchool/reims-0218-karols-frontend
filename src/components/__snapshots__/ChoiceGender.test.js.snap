// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="mb-4"
>
  <div
    className="card"
  >
    <div
      className="wrapper text-center"
      style={
        Object {
          "backgroundImage": "url(undefined)",
          "backgroundPosition": "center",
          "backgroundRepeat": "no-repeat",
          "backgroundSize": "cover",
        }
      }
    >
      <div
        className="data"
      >
        <div
          className="content card-body"
        >
          <h5
            className="city card-title"
            style={
              Object {
                "fontWeight": "500",
                "marginTop": "10px",
              }
            }
          >
            <p
              style={
                Object {
                  "color": "#FFF",
                }
              }
            />
          </h5>
          <button
            className="btn btn-secondary"
            onClick={[Function]}
            style={
              Object {
                "color": "#fff",
                "display": "block",
                "fontSize": "12px",
                "fontWeight": "700",
                "lineHeight": "1",
                "margin": "2em auto 1em",
                "marginBottom": "40px",
                "padding": "10px 20px",
                "position": "relative",
                "textAlign": "center",
              }
            }
            type={undefined}
          >
            SÃ©lectionner
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
